_drafts/making-backend-scalable.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Making Autolab's Backend Scalable"
+author: Ilter Canberk
+
+tags: [autolab, community, backend, scalability]
```

Comment:
I would say that fewer tags is better, and also that tagging specific things is better. I would probably tag this just Tango, Redis, and maybe scalability .